dev/com.ibm.ws.microprofile.metrics.3.0/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -0,0 +1,140 @@
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWMMC
+#COMPONENTNAMEFOR CWMMC MicroProfile Metrics API 
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#ISMESSAGEFILE true
+# #########################################################################
+###############################################################################
+# Copyright (c) 2009, 20120 IBM Corporation and others.
```

Comment:
Fix the year

Reply:
Fixed in https://github.com/OpenLiberty/open-liberty/pull/12299/commits/9edbbeb2b39fae1a4d57c5f9a9c65443ee9ffbc9